test/nlp/utils.jl

```diff
@@ -19,7 +19,7 @@ end
 
 @testset "Increase coverage of default_NLPcounters" begin
   @default_counters SuperNLPModel model
-  nlp = SuperNLPModel(SimpleNLPModel())
+  nlp = SuperNLPModel{Float64, Vector{Float64}}(SimpleNLPModel())
```

Comment:
It might be useful to also have a test involving another type than the default `Float64`.